docs/src/main/sphinx/release/release-390.md

```diff
@@ -0,0 +1,59 @@
+# Release 390 (13 July 2022)
+
+## General
+
+* Update minimum required Java version to 17.0.3. ({issue}`13014`)
+* Add support for [setting comments on views](/sql/comment). ({issue}`8349`) ({issue}`8349`)
+* Improve performance of queries with an `UNNEST` clause. ({issue}`10506`)
+* Fix potential query failure when spilling to disk is enabled by the
+  `force-spilling-join-operator` configuration property or the
+  `force_spilling_join` session property. ({issue}`13123`)
+* Fix incorrect results for certain join queries containing filters involving
+  explicit or implicit casts. ({issue}`13145 `)
+
+## Cassandra connector
+
+* Change mapping for Cassandra `inet` type to Trino `ipaddress` type.
+  Previously, `inet` was mapped to `varchar`. ({issue}`851`)
+* Remove the unused configuration properties
+  `cassandra.load-policy.use-token-aware`,
+  `cassandra.load-policy.shuffle-replicas`, and
+  `cassandra.load-policy.allowed-addresses`. ({issue}`12223`)
+
+## Delta Lake connector
+
+* Add support for filtering splits based on `$path` column predicates. ({issue}`13169`)
+* Expose AWS Glue metastore statistics via JMX. ({issue}`13087`)
+* Fix failure when using the Glue metastore and queries contain `IS NULL` or
+  `IS NOT NULL` filters on numeric partition columns. ({issue}`13124`)
+
+## Hive connector
+
+* Expose AWS Glue metastore statistics via JMX. ({issue}`13087`)
+* Add support for [setting comments on views](/sql/comment). ({issue}`13147`)
+* Add support for Databricks runtime 10.4 LTS. ({issue}`13081`)
```

Comment:
Should that not be in the Delta lake connector as well or instead? Also .. is this for metastore and Delta Lake table format somehow?